docs/getting_started/quickstart.rst

```diff
@@ -0,0 +1,165 @@
+Getting Started
+===================
+
+.. note:: 
+  | *Estimated Reading Time: 10 minutes*
+  | You can follow along this tutorial in a Jupyter notebook `here <https://github.com/modin-project/modin/tree/master/examples/tutorial/tutorial_notebooks/introduction/quickstart.ipynb>`_. 
+
+.. toctree::
+    :hidden:
+    :maxdepth: 4
+    
+    installation
+    using_modin/using_modin_locally
+    using_modin/using_modin_cluster
+    why_modin/pandas
+    why_modin/out_of_core
+    why_modin/modin_vs_dask
+    examples
+    faq
+    troubleshooting
+
+.. meta::
+    :description lang=en:
+        Introduction to Modin.
+
+Quick Start Guide
+----------------
+
+To install the most recent stable release for Modin run the following:
+
+.. code-block:: bash
+
+  pip install modin[all] 
+
+For further instructions on how to install Modin with conda or for specific platforms or engines, see our detailed `installation guide <../getting_started/installation.html>`_.
+
+Modin acts as a drop-in replacement for pandas so you can simply change a single line of import to speed up your pandas workflows. To use Modin, you simply have to replace the import of pandas with the import of Modin, as follows.
+
+.. code-block:: bash
+
+  # import pandas as pd
+  import modin.pandas as pd
+
+Example: Instant Scalability with No Extra Effort
+--------------------------------------------------
+
+When working on large datasets, pandas becomes painfully slow or `runs out of memory <../why_modin/out_of_core.html>`_. Modin automatically scales up your pandas workflows by parallelizing the dataframe operations, so that you can more effectively leverage the compute resource available.
+
+For the purpose of demonstration, we will load in modin as ``pd`` and pandas as ``old_pd``.
+
+.. code-block:: python
+
+  import modin.pandas as pd
+  import pandas as old_pd
```

Comment:
As I remember we agreed to use just pandas, but not old_pd.

Reply:
Changed to `import pandas` and replaced `old_pd` throughout